src/Commands/SetLifter.cpp

```diff
@@ -0,0 +1,41 @@
+#include "SetLifter.h"
+
+SetLifter::SetLifter(Shooter::LifterValue)
+{
+	// Use Requires() here to declare subsystem dependencies
+	// eg. Requires(chassis);
+}
+
+// Called just before this Command runs the first time
+void SetLifter::Initialize()
+{
+
+}
+
+// Called repeatedly when this Command is scheduled to run
+void SetLifter::Execute()
+{
+Shooter::LifterValue
+	= Shooter::kUp ? Robot::shooter->SetLifter(Shooter::kUp) : Robot::shooter->SetLifter(Shooter::kUp);
```

Comment:
No need to evaluate the value here. You're passing the value with the same type straight to the shooter.  Just use

`Robot::shooter(mShooterValue);`